package.json

```diff
@@ -4,6 +4,7 @@
     "scripts": {
         "start": "npm run dev",
         "dev": "vite --port 8080",
+        "dev-host": "npm run dev -- --host --cors",
```

Comment:
I think we could `--host --cors` on `npm run dev` directly, it is a target solely used for local development, and we should often check on our real phones how it looks like